x-pack/plugins/apm/public/components/app/backend_detail_overview/backend_latency_chart.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { getDurationFormatter } from '../../../../common/utils/formatters';
+import { useApmBackendContext } from '../../../context/apm_backend/use_apm_backend_context';
+import { useUrlParams } from '../../../context/url_params_context/use_url_params';
+import { useComparison } from '../../../hooks/use_comparison';
+import { useFetcher } from '../../../hooks/use_fetcher';
+import { useTimeRange } from '../../../hooks/use_time_range';
+import { Coordinate, TimeSeries } from '../../../../typings/timeseries';
+import { TimeseriesChart } from '../../shared/charts/timeseries_chart';
+import { useTheme } from '../../../hooks/use_theme';
+import {
+  getMaxY,
+  getResponseTimeTickFormatter,
+} from '../../shared/charts/transaction_charts/helper';
+
+export function BackendLatencyChart({ height }: { height: number }) {
+  const { backendName } = useApmBackendContext();
+
+  const theme = useTheme();
+
+  const { start, end } = useTimeRange();
+
+  const {
+    urlParams: { kuery, environment },
+  } = useUrlParams();
+
+  const { offset, comparisonChartTheme } = useComparison();
+
+  const { data, status } = useFetcher(
+    (callApmApi) => {
+      if (!start || !end) {
+        return;
+      }
+
+      return callApmApi({
+        endpoint: 'GET /api/apm/backends/{backendName}/charts/latency',
+        params: {
+          path: {
+            backendName,
+          },
+          query: {
+            start,
+            end,
+            offset,
```

Comment:
I figure `offset` is a simpler API than `comparisonStart` + `comparisonEnd`, but open to feedback here.

Reply:
I feel like if this was changed we should update the others APIs too. Otherwise now we'll have two ways to get comparison data.